src/solver/optimisation/include/antares/solver/optimisation/opt_structure_probleme_a_resoudre.h

```diff
@@ -23,24 +23,30 @@
 
 #include <vector>
 #include <string>
+#include <memory>
 #include "opt_constants.h"
-#include "ortools/linear_solver/linear_solver.h"
 
 /*--------------------------------------------------------------------------------------*/
 
-namespace Antares::Solver::Optimization {
-
+namespace Antares::Solver::Optimization
+{
 struct OptimizationOptions
 {
     bool useOrtools;
     std::string solverName;
 };
+} // namespace Antares::Solver::Optimization
 
+namespace Antares::Optimization
+{
+struct BasisStatus;
 }
 
 /* Le probleme a resoudre */
 struct PROBLEME_ANTARES_A_RESOUDRE
 {
+    PROBLEME_ANTARES_A_RESOUDRE();
```

Comment:
The type is now not copyable / not movable: if this is what we want, we should make it explicit by at least deleting copy constructor and assignment, see https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types for background